action.yml

```diff
@@ -29,6 +29,10 @@ inputs:
     description: "Profile configuration"
     required: false
     default: profile.yaml
+  timings:
```

Comment:
would it be good to add the new `level` flag as well?

Reply:
df43ff6

Reply:
Still needs testing as this is not testable until 0.9.0 image is released.